src/test/javascript/TexasPick3WebScraperUnitTest.js

```diff
@@ -64,4 +64,22 @@ describe("Cheerio tests", function() {
 
         assert.equal(actualEveningWinningNumber, expectedEveningWinningNumber);
     });
+    it("should be able to find Night Winning Number for a specific date", function() {
+        var html = fs.readFileSync(fixturePath + "pick3-morning-drawing-fixture.html", "UTF-8"),
+            expectedDate = "02/14/2018",
```

Comment:
Lets change expectedDate variable to actualDrawDate variable name.  Please make similar change to all other tests.

Reply:
Changed